tests/test_config.py

```diff
@@ -514,35 +514,32 @@ def test_discovering_configuration_auto_detect_fails(self, mock_detect,
         assert len(self.hass.config.whitelist_external_dirs) == 1
         assert "/test/config/www" in self.hass.config.whitelist_external_dirs
 
-    @mock.patch('asyncio.create_subprocess_exec')
-    def test_check_ha_config_file_correct(self, mock_create):
+    @mock.patch('os.path.isfile', return_value=True)
+    def test_check_ha_config_file_correct(self, mock_file):
```

Comment:
We already have tests for `check_ha_config_file` so we should stub that out here and just have it return a dummy error log.

Reply:
We should try to do 0 I/O inside our tests. I/O in tests make them unreliable and slow (especially on Travis).